Winapp2.ini

```diff
@@ -5203,6 +5203,13 @@ RegKey5=HKCU\Software\Corel\CorelDRAW\17.0\Draw\Application Preferences\HTML Exp
 RegKey6=HKCU\Software\Corel\CorelDRAW\17.0\Draw\Application Preferences\NewFromTemplate|BrowseTemplateDir
 RegKey7=HKCU\Software\Corel\CorelDRAW\17.0\PHOTO-PAINT\Application Preferences\Framework|RecentFiles
 
+[Corsair iCue Logs *]
+LangSecRef=3024
+DetectFile=%CommonAppData%\Corsair\CUE\Service logs\*.log
```

Comment:
Should we avoid volatile entries? (Entries that delete their detect parameters)

Reply:
Yes, that would be much better for people using the "Trim" function.

Before merging the pull request, I can change the Detect to:
DetectFile=%CommonAppData%\\Corsair\\CUE\\Service logs

(BTW: There are a few more volatile entries in winapp2.ini, but as far as I know most of them can't be fixed without checking/installing the programs.)